libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -55,27 +67,11 @@
 #define MQTT_PACKET_TYPE_PINGRESP       ( ( uint8_t ) 0xD0U )  /**< @brief PINGRESP (server-to-client). */
 #define MQTT_PACKET_TYPE_DISCONNECT     ( ( uint8_t ) 0xE0U )  /**< @brief DISCONNECT (client-to-server). */
 
-
 /**
  * @brief The size of MQTT PUBACK, PUBREC, PUBREL, and PUBCOMP packets, per MQTT spec.
  */
 #define MQTT_PUBLISH_ACK_PACKET_SIZE    ( 4UL )
 
-struct MQTTFixedBuffer;
```

Comment:
Doxygen does not easily support forward declarations like this, so i revert to our original style of `typedef struct mystrcyt {} mystruct_t;`